ci/pipeline.yaml

```
@@ -354,52 +357,42 @@ jobs:
           image_resource:
             type: registry-image
             source:
-              repository: karlkfi/concourse-dcind
+              repository: registry.access.redhat.com/ubi8/ubi-minimal
           inputs:
-            - name: agent-operator-release-source
-          outputs:
-            - name: message
-            - name: latest-operator-image-manifest-sha
-            - name: docker-cfg
+            - name: agent-operator-image-manifest-sha
           params:
-            DOCKER_CLI_EXPERIMENTAL: enabled
-            COMMIT_SHA: ((.:commit-sha))
+            GIT_COMMIT: ((.:git-commit))
             VERSION: ((.:operator-version))
-            JSON_KEY: ((project-berlin-tests-gcp-instana-qa))
             RED_HAT_REGISTRY_PASSWORD: ((redhat-container-registry-5e961c2c93604e02afa9ebdf.password))
             RED_HAT_REGISTRY_USERNAME: ((redhat-container-registry-5e961c2c93604e02afa9ebdf.user))
             ICR_PASSWORD: ((concourse-icr-containers-public.password))
             ICR_USERNAME: iamapikey
+            DELIVERY_INSTANA_IO_USER: ((delivery-instana-io-internal-project-artifact-read-writer-creds.username))
+            DELIVERY_INSTANA_IO_PW: ((delivery-instana-io-internal-project-artifact-read-writer-creds.password))
```

This is just a suggestion to match the naming in the other pipelines
```suggestion
            ARTIFACTORY_USERNAME_INTERNAL: ((delivery-instana-io-internal-project-artifact-read-writer-creds.username))
            ARTIFACTORY_PASSWORD_INTERNAL: ((delivery-instana-io-internal-project-artifact-read-writer-creds.password))
```